TheMovieDb/Components/CircularProgressBarView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CircularProgressBarView.swift
+//  TheMovieDb
+//
+//  Created by Juan David Torres on 30/10/21.
+//
+
+import UIKit
+
+protocol CircularProgressBarViewP {
```

Comment:
the P at the end provides no clarity maybe use Protocol or instead you can use something that represents better the idea sorta like `ProgressViewCircularEvents`